### DISCUSSION TITLE
Releasing new version on crates.io with support for hdf5 1.14?

### DISCUSSION THREAD

**Discussion by yuyttenhove** [aldanor/hdf5-rust#253] [Questions]
Hi,

I'm sorry if this has been answered somewhere already, but is there something holding back releasing a new version on crates.io with support for hdf5 versions > 1.12? 
I see that the latest release was dates back to november 21st 2021 and hdf5 1.14 has been released for over seven months now.

Many thanks, Yolan

**Comment by mulimoen** [comment]
It is currently stalled on a compilation error in hdf5 for mingw/wine https://github.com/aldanor/hdf5-rust/pull/243

**Comment by jedbrown** [comment]
This is causing frequent issues now that hdf5-1.14 is widespread in Linux distributions, causing `hdf5-sys` to fail. At a glance, the above issue looks related to `hdf5-src`, but maybe a new `hdf5-sys` release can still be made on crates.io?
